ibm/service/secretsmanager/resource_ibm_sm_private_certificate_configuration_intermediate_ca.go

```diff
@@ -737,9 +859,56 @@ func resourceIbmSmPrivateCertificateConfigurationIntermediateCAMapToConfiguratio
 		}
 		model.PostalCode = postalCode
 	}
+	if _, ok := d.GetOk("crypto_key"); ok {
+		CryptoKeyModel, err := resourceIbmSmPrivateCertificateConfigurationMapToPrivateCertificateConfigurationCryptoKey(d.Get("crypto_key").([]interface{})[0].(map[string]interface{}))
```

Comment:
Accessing d.Get("crypto_key").([]interface{})[0] without length check will be panic if users just provides crypto_key {} since none of inner element is required. Add a check for length and access

Reply:
Same comment applies for down resources also

Reply:
@hkantare fixed